### DISCUSSION TITLE
Unable to Create "web" Directory for Drupal Projects in Docker with Composer 2.7.0

### DISCUSSION THREAD

**Discussion by daniela-zc** [composer/composer#11853] [General]
I am not sure if this is the right place to report something like this, but it would be awesome to know a different workaround.

Issue Report:
When attempting to install Drupal into a Docker container using Composer version 2.7.0, the expected "web" directory is missing, causing issues with project setup. This behavior is unexpected and disrupts the standard Drupal installation process within Docker environments.

Temporal solution:
Downgrade to Composer 2.6.6: As a temporary workaround, reverting to Composer version 2.6.6 has been observed to resolve the issue. This allows for the creation of the "web" directory as expected, enabling the Drupal project setup within Docker containers.

**Comment by fredden** [comment]
This looks like a duplicate of https://github.com/composer/composer/issues/11854
